.github/workflows/node-ci.yml

```diff
@@ -86,8 +119,44 @@ jobs:
           path: ./node_modules
           key: modules-${{ hashFiles('package-lock.json') }}
 
-      - name: Create production build
-        run: npm run build
+      - name: Download dist directory
+        uses: actions/download-artifact@v4
+        with:
+          name: dist
+          path: dist
 
       - name: Validate HTML
         run: npm run lint:html
+
+  unit-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    needs: [setup, build]
```

Comment:
Why do tests need a full build? Is it because you need the files generated in `src/lib/`? Would `npm run prep` on `pretest` or `pretest:unit` suffice?

Reply:
This also works, so I changed it to this!